torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -0,0 +1,157 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import * as urllib from "urllib";
+
+import { getOctokit } from "lib/github";
+import fetchFlakyTests from "lib/fetchFlakyTests";
+import { FlakyTestData, IssueData } from "lib/types";
+import { supportedPlatforms } from "lib/bot/verifyDisableTestIssueBot";
+import fetchIssuesByLabel from "lib/fetchIssuesByLabel";
+
+
+const NUM_HOURS = 3;
+const owner: string = "pytorch";
+const repo: string = "pytorch";
+
+
+export default async function handler(
+    req: NextApiRequest,
+    res: NextApiResponse<void>
+) {
+    await disableFlakyTests();
```

Comment:
Back to my OG question--how can we prevent someone from using up all our tokens by spamming this endpoint?

Reply:
the `req` object will hold headers; you can check the bearer token we're passing from the GitHub action and 403 if it's wrong.

Reply:
we'll probably have to implement it here https://nextjs.org/docs/authentication. Maybe the next auth is suitable and we can use the github auth/app or something? I've never implemented it befor.

Reply:
I think it's probably fine to just add the bearer token value to our vercel env and just do an equality check. We don't really need fancy auth here

Reply:
Okay, glad a simple equal is cool. Will add that :)